js/widgets/page.js

```diff
@@ -46,6 +46,7 @@ $.widget( "mobile.page", {
 
 		// Deprecated in 1.4 remove in 1.5
 		contentTheme: null,
+		enhanceWithin: true,
```

Comment:
I agree. I just went with some option name, but, of course, the actual name is up for discussion.

- [ ] autoEnhance?
- [ ] enhanceChildren?
- [ ] enhance?
- [ ] autoinit?
- [ ] autoinitChildren?